frontend/src/__tests__/events/match/handleWinRoundsNumberUpdate.test.ts

```diff
@@ -0,0 +1,173 @@
+
+import '../setupTests';
+import { Game, Match, Round } from '@/interfaces/Lobby';
+import { handleWinRoundsNumberUpdate } from '@/events/match/handlers/winRoundsNumberUpdate';
+import { i18n } from '@/plugins/i18n';
+import { lobby } from '@/connection';
+
+describe('handleWinRoundsNumberUpdate', () => {
+  const leaderPlayer = {
+    id: '12345',
+    name: 'leader player',
+    leader: true,
+    ready: true
+  };
+
+  const anotherPlayer = {
+    id: '123',
+    name: 'John joe',
+    leader: false,
+    ready: false
+  };
+
+  const EXPECTED_PLAYER_NUM_WONS_ROUNDS = 1;
+  const EXPECTED_NEXT_PLAYER_ID = anotherPlayer.id;
+
+  test('Deve atual', () => {
```

Comment:
Finalizar a descrição do caso de teste

**Sugestão:**
```suggestion
  test('Deve atualizar corretamente o número de rodadas que o jogador atual acredita que irá vencer', () => {
```